app/channels/application_cable/connection.rb

```diff
@@ -22,11 +22,8 @@ def find_verified_user
     end
 
     def find_exercise
-      if (current_exercise = Exercise.find(request.params[:exercise_id]))
-        current_exercise
-      else
-        raise 'Exercise was somehow not found'
-      end
+      raise 'Exercise was somehow not found' unless (current_exercise = Exercise.find(request.params[:exercise_id]))
```

Comment:
Is raising the exception necessary here since `Exercise#find` raises an exception when exercise isn't found?